crates/bevy_app/src/main_schedule.rs

```diff
@@ -181,6 +181,10 @@ pub struct PostUpdate;
 #[derive(ScheduleLabel, Clone, Debug, PartialEq, Eq, Hash)]
 pub struct Last;
 
+/// Animation system set. This exists in [`PostUpdate`].
```

Comment:
Fascinating choice. I've been reluctant to clutter this file with this sort of interop / glue, but I think that this is a good way to handle this flavor of indirect ordering depdendency.

Reply:
Yeah. "animation" feels like a general enough concept to merit this kind of "concept promotion". I think we should be stingy about what we allow here, but promoting "arbitrary animation happens in this part of PostUpdate" feels reasonable to me, especially given that it makes our compile graph nicer. I think the more opinionated something is, the more we should discourage "promotion".